src/modules/admin/components/AdminDashboard/AdminDashboard.tsx

```diff
@@ -152,11 +161,12 @@ const AdminDashboard = ({
     [colonyName],
   );
 
-  const { error: colonyError, data: colony } = useDataSubscriber(
-    colonySubscriber,
-    [colonyAddress],
-    [colonyAddress],
-  );
+  const {
+    data: { colony } = {},
+    loading: colonyDataLoading,
```

Comment:
loading isn't really necessary here. We can just rely on the fact that there is data there.